Engine/UI/DebugDraw.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _DEBUGDRAW_H_
```

Comment:
Is this file only for defining types?

Reply:
It's used as a wrapper of `debug_draw.hpp`. It converts `MathGeoLib` types to required ones and also serves as an entry point to the library.